dashboard/app/controllers/api/v1/pd/applications_controller.rb

```diff
@@ -30,7 +30,14 @@ def index
 
   # GET /api/v1/pd/applications/1
   def show
-    render json: @application, serializer: Api::V1::Pd::ApplicationSerializer
+    render json: @application, each_serializer: Api::V1::Pd::ApplicationSerializer
```

Comment:
Why `each_serializer`? Isn't this a single application?

Reply:
oh good catch, I did not intend to change that.